modules/javaparser/3.26.2/patches/build_with_bazel.patch

```diff
@@ -0,0 +1,76 @@
+--- /dev/null
++++ BUILD.bazel
+@@ -0,0 +1 @@
++
+
+--- /dev/null
++++ javaparser-core/BUILD.bazel
+@@ -0,0 +1,32 @@
++load(":gen_java_parser.bzl", "gen_java_parser")
++
++gen_java_parser(
++    name = "javacc_java_jj",
++    src = "src/main/javacc/java.jj",
++    outs = [
++        "GeneratedJavaParser.java",
++        "GeneratedJavaParserConstants.java",
++        "GeneratedJavaParserTokenManager.java",
++        "ParseException.java",
++        "Provider.java",
++        "SimpleCharStream.java",
++        "StreamProvider.java",
++        "StringProvider.java",
++        "Token.java",
++        "TokenMgrException.java",
++    ],
++    out_dir = "src/main/java/com/github/javaparser",
++)
++
++java_library(
++    name = "javaparser-core",
```

Comment:
This project has four public libraries: https://mvnrepository.com/artifact/com.github.javaparser
- `javaparser-core`
- `javaparser-core-serialization`
- `javaparser-symbol-solver-core`
- `java-symbol-solver-core`

I'm only writing a `java_library` wrapper for `javaparser-core` because that's the one I need at the moment, but a future revision of the module might wrap the others if anyone needs that functionality.